webrtc.html

```diff
@@ -4712,9 +4712,13 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             <dt><dfn><code>codecs</code></dfn> of type <span class=
             "idlMemberType">sequence&lt;<a>RTCRtpCodecParameters</a>&gt;</span></dt>
             <dd>
-              <p>A sequence containing the codecs that an
-              <a><code>RTCRtpSender</code></a> will choose from in order to
-              send media.</p>
+            <p>
+              A sequence containing the codecs (including RTX/RED/FEC) that
+              an <a><code>RTCRtpSender</code></a> will choose from in order
+              to send media. There will be an "rtx" entry in <code>codecs[]</code>
+              for each media codec that can be retransmitted, each with their own
```

Comment:
English nit: A codec isn't retransmitted. "for each media codec where retransmission via rtx can be used"?
It's still not clear - if VP8 and H264 are supported, each with RTX, are there 4 elements, what are the names of the last two, and how do we tell them apart (if we need to)? An example may be useful.